### PR TITLE
Deleting Connections Functional

### PR DESCRIPTION
**Pull request by hjdjoo**
WHEN a user selects clusters from the "Connections" page and clicks on the delete button,
THEN the clusters are identified by ID and removed from the database
AND THEN the clusters are removed from the configuration files
AND THEN any unused configuration files are deleted from the root directory
AND THEN docker containers are restarted, with orphan containers removed.
AND IF all clusters are removed,
THEN the docker-compose file should have no depends_on property under grafana.

### CODE REVIEW

File: .ssl_certificates/localhost+2-key.pem
Comment (echochs1) [inline .ssl_certificates/localhost+2-key.pem:1]: Should be removed

File: .ssl_certificates/localhost+2.pem
Comment (echochs1) [inline .ssl_certificates/localhost+2.pem:1]: remove

File: docker-compose.yml [hunk 1]
```diff
@@ -3,41 +3,23 @@ services:
...
-      - ./kafka_controller_alerts.yml:/etc/prometheus/kafka_controller_alerts.yml:ro
-      - prometheus_data:/prometheus
-    ports:
-      - '9090:9090'
+    - '3000:3000'
   alertmanager:
     image: prom/alertmanager:latest
     ports:
-      - '9094:9094'
+    - '6093:6093'
```
Comment (echochs1) [inline docker-compose.yml:19]: why 6093?

File: main/controllers/dbController.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+dbController.deleteConnections = async (req, res, next) => {
+
+  try {
+    // req.body should contain two pieces of information: userid and the array of ports.
+    const { userid, clusters } = req.body;
+    // we'll first delete those particular rows from the database
+
+    const clustersQuery = clusters.map((node, index) => {
+      return `$${index + 2}`
```
Comment (echochs1) [inline main/controllers/dbController.js:60]: why + 2?

File: src/components/ConnectionsTable.jsx
Comment (echochs1) [inline src/components/ConnectionsTable.jsx:1]: why was the component renamed?

File: docker-compose.yml (hunk 1 above)
Comment (hjdjoo) [reply docker-compose.yml:19]: Just needed to set it to something that wouldn't create port conflicts as we add new instances of prometheus. It was listening on port 9093 before so prometheus #4 would always wind up bailing.

File: src/components/ConnectionsTable.jsx
Comment (hjdjoo) [reply src/components/ConnectionsTable.jsx:1]: I don't recall changing any component names -- where is that showing up?

File: main/controllers/dbController.js (hunk 2 above)
Comment (hjdjoo) [reply main/controllers/dbController.js:60]: Clusters is an array of ID numbers, but I needed them to be mapped to the query variables $2, $3, $4, etc. UserID is always going to be $1. So index 0 becomes $2, index 1 becomes $3, etc.

File: docker-compose.yml (hunk 1 above)
Comment (hjdjoo) [reply docker-compose.yml:19]: Also why 430AM

**Review by echochs1 — APPROVED** [review]

**Pull request merged**